helix-core/src/test/java/org/apache/helix/TestHelper.java

```diff
@@ -287,7 +289,10 @@ public static void setupCluster(String clusterName, String zkAddr, int startPort
 
       for (int i = 0; i < resourceNb; i++) {
         String resourceName = resourceNamePrefix + i;
-        setupTool.addResourceToCluster(clusterName, resourceName, partitionNb, stateModelDef, mode.toString());
+        setupTool.addResourceToCluster(clusterName, resourceName, partitionNb, stateModelDef,
+            mode.name(),
+            mode == RebalanceMode.FULL_AUTO ? CrushEdRebalanceStrategy.class.getName()
+                : RebalanceStrategy.DEFAULT_REBALANCE_STRATEGY);
```

Comment:
Here, RebalanceMode.FULL_AUTO, we will always use crushED, which means delayAutoRebalancer with CrushED

Before this change, it is AutoRebalancer with AutoRebalanceStrategy, both takes current state into consideration. They bestPossibleVerifier may still work, but the state mapping can be different from one run to another run